examples/onboarding/src/pages/NewEmployee.tsx

```diff
@@ -0,0 +1,88 @@
+import { Button } from "@/components/Button.tsx";
+import { NavigateBack } from "@/components/NavigateBack.tsx";
+import { Stack } from "@/components/Stack.tsx";
+import { TextInput } from "@/components/TextInput.tsx";
+import { useAccount, useCoState } from "@/main.tsx";
+import { Group, ID } from "jazz-tools";
+import { useCallback, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import {
+  CoDocUploadStep,
+  CoEmployee,
+  CoFinalStep,
+  CoInitialStep,
+  EmployeeCoList,
+} from "../schema.ts";
+
+export function NewEmployee({
+  employeeListCoId,
+}: {
+  employeeListCoId: ID<EmployeeCoList>;
+}) {
+  const navigate = useNavigate();
+  const { me } = useAccount();
+  const writerGroup = Group.create({ owner: me });
+  const readerGroup = Group.create({ owner: me });
+  readerGroup.addMember("everyone", "reader");
```

Comment:
**nit:** having these "Group.create" here means that we do create these groups at each render.

Since we use them only inside `createEmployee` I would suggest to move the groups cration inside there.

Reply:
Well spotted! fixed that.